arangod/VocBase/LogicalCollection.h

```diff
@@ -122,8 +122,6 @@ class LogicalCollection : public LogicalDataSource {
 
   TRI_col_type_e type() const;
 
-  std::string globallyUniqueId() const;
```

Comment:
don't we need it somewhere?

Reply:
No, and btw we don't have definition this function, so I think its merge/etc artifact in devel